src/back-in-stock.plugin.ts

```diff
@@ -40,66 +31,6 @@ export interface BackInStockOptions {
     allowSubscriptionWithoutSession?: boolean;
 }
 
-/**
- * @description
- * Subscribes to {@link ProductVariant} inventory changes
- * and FIFO updates BackInStock {@link BackInStock} to be notified
- * to the amount of saleable stock with plugin init option
- * limitEmailToStock = true or false to notify all subscribers
- *
- */
-@Injectable()
-@EventSubscriber()
-export class ProductVariantSubscriber implements EntitySubscriberInterface<ProductVariant> {
```

Comment:
`ProductVariantSubscriber` has been replaced by `eventBus.ofType(StockMovementEvent)`, because in V2 multiple entities (StockLocations) can be responsible for the final `saleableStock`